services/cli/src/bencher/sub/project/run/ci/github_actions.rs

```diff
@@ -244,6 +253,43 @@ impl GitHubActions {
 
         Ok(())
     }
+
+    async fn create_github_check(
+        &self,
+        report_comment: &ReportComment,
+        event_str: &str,
+        event: &serde_json::Value,
+    ) -> Result<CheckRun, GitHubError> {
+        let full_name = repository_full_name(event_str, event)?;
+        let (owner, repo) = split_full_name(full_name)?;
+
+        let report = CheckRunOutput {
+            title: "🐰 Bencher Report".to_owned(),
```

Comment:
This needs to be a hyperlink to the Report. So maybe try reverting this and just putting an empty string here?

Reply:
Let's try empty string (ping @dklassic), but if it does not work, this is the best we can get (title is escaped so no md/html is possible).

Reply:
[Tested and yeah it works](https://github.com/dklassic/bencher-test/actions/runs/11497758338/job/32008653984).

Reply:
Empty title still causes empty `<h2>` to be created, which causes another _______ line:
![slika](https://github.com/user-attachments/assets/43e378b1-47f9-4788-8293-15a70663d504)

@epompeii what do you prefer?

Reply:
@sagudev I would prefer the extra, useless `<h2>` and it's ensuing extra `<hr />`.

So yes, please revert that commit and let's just move over to using the empty string.

Reply:
Sorry for late response, I got this notification in my spam folder 😞.